Renaming/persistent sampling

jmlarson1 opened the pull request.

shuds13 commented:
Might want to consider making the function names more consistent. I dropped the persistent_ prefix from the functions I added as its already in the module name.

jmlarson1 commented:
> Might want to consider making the function names more consistent. I dropped the persistent_ prefix from the functions I added as its already in the module name.

I'm not sure. This could be a larger renaming if desired (in a future PR).
